lib/question.js

```diff
@@ -1011,14 +1012,57 @@ module.exports = {
 
                 const studentMessage = 'Error rendering question';
                 const courseData = {variant, question, submission, course};
-                this._writeCourseIssues(null, courseIssues, variant.id, locals.authn_user.user_id, studentMessage, courseData, (err) => {
+                // locals.authn_user may not be populated when rendering a panel
+                const user_id = (locals && locals.authn_user) ? locals.authn_user.user_id : null;
```

Comment:
This is equivalent to `_.get(locals, 'authn_user.user_id', null)` if you prefer not to code it yourself (I'm not convinced lodash is worth it here).